en/docs/install-and-setup/install-and-setup-overview.md

```diff
@@ -0,0 +1,23 @@
+# Install and Setup Overview
```

Comment:
Should this page be in here?

Reply:
Yes. Previously, the wrong version of the page was committed.